packages/snage/src/config/parser.test.ts

```diff
@@ -26,4 +26,56 @@ describe('parseConfig', () => {
             })
         );
     });
+    it('fieldname containing non existing fields is invalid', () => {
```

Comment:
```suggestion
    it('fails on not existing fields in fieldName', () => {
```
Die anderen auch noch verbessern.